assets/src/editor/blocks/donation-bar/index.js

```diff
@@ -0,0 +1,254 @@
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/**
+ * Block for sharing an article on Twitter, Facebook, LinkedIn, or via Email, with a Copy Link option.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { registerBlockType } from '@wordpress/blocks';
+import { useBlockProps, InspectorControls } from '@wordpress/block-editor';
+import { __ } from '@wordpress/i18n';
+import ServerSideRender from '@wordpress/server-side-render';
+import {
+	PanelBody,
+	TextControl,
+	ToggleControl,
+	Button,
+	Flex,
+	FlexItem,
+	SelectControl
+} from '@wordpress/components';
+
+import metadata from './block.json';
+
+import './index.css';
+
+const supportedCurrencies = [
+	{ label: 'USD ($)', value: 'USD' },
+	{ label: 'CAD (CA$)', value: 'CAD' },
```

Comment:
Could we localize this array from PHP, since we define it in geoip on the backend? Feels like a foot gun to have to maintain it two places

Reply:
Updated, cheers for the push.